run_conode.sh

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env bash
+set -e
+
+VERSION=1.0-pre1
```

Comment:
Well, we should have a consistent versionning system, putting at two places seems like trouble for the future...